detection_rules/docs.py

```diff
@@ -546,3 +548,458 @@ def name_to_title(name: str) -> str:
     """Convert a rule name to tile."""
     initial = re.sub(r'[^\w]|_', r'-', name.lower().strip())
     return re.sub(r'-{2,}', '-', initial).strip('-')
+
+
+@dataclass
+class UpdateEntry:
+    """A class schema for downloadable update entries."""
+    update_version: str
+    date: str
+    new_rules: int
+    updated_rules: int
+    note: str
+    url: str
```

Comment:
Nit. Should these be in alphabetical order? Possibly not if it impacts order of the generated docs, but it looks like the order is determined with a formater separate from the initial formatting of the dataclass.

Reply:
That's right. It's the order of the entries.